src/js/pages/TeamHome.jsx

```diff
@@ -33,6 +33,7 @@ const TeamHome = ({ classes }) => {
 
   // const isAddPersonDrawerOpen = document.getElementById('addPersonDrawer');
 
+  // TODO is this even used?
```

Comment:
We need person-list-retrieve on src/js/pages/TeamHome for cases where a person lands directly on the TeamHome page as the first page.